examples/complete/outputs.tf

```
@@ -1,69 +1,69 @@
 output "role_name" {
-  value       = module.role.name
+  value       = module.role.name != "" ? module.role.name : null
```

We very much should not be modifying the outputs of the module under test, because that hides errors.

```suggestion
  value       = module.role.name
```